README.md

```diff
@@ -1410,6 +1410,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
    * [gogeo.io](https://gogeo.io/) — Maps and geospatial services with an easy to use API and support for big data.
    * [graphhopper.com](https://www.graphhopper.com/) A free package for developers is offered for Routing, Route Optimization, Distance Matrix, Geocoding, Map Matching.
    * [here](https://developer.here.com/) — APIs and SDKs for maps and location-aware apps. 250k transactions/month for free.
+   * [locationiq.com] (https://locationiq.com/) — Geocoding, Maps and Routing APIs. 5000 requests/day for free.
```

Comment:
nice, please fix the markdown syntax here otherwise good

Reply:
Fixed, thanks!